packages/stream_chat_flutter/lib/src/attachment/giphy_attachment.dart

```diff
@@ -1,6 +1,9 @@
+import 'dart:typed_data';
+
 import 'package:cached_network_image/cached_network_image.dart';
 import 'package:flutter/material.dart';
 import 'package:stream_chat_flutter_core/stream_chat_flutter_core.dart';
+import 'package:transparent_image/transparent_image.dart';
 
 import '../full_screen_media.dart';
```

Comment:
May want to change these to package imports, one less thing to change in our lints 😉

Reply:
I don't think we should do this 
I'm the first one who would do it because I like spaghetti and everything similar (spaghetti code, spaghetti repo and so on...), but generally speaking I don't think it's a good practice